src/net_processing.cpp

```diff
@@ -657,7 +657,7 @@ class PeerManagerImpl final : public PeerManager
 
     /** Whether we've completed initial sync yet, for determining when to turn
       * on extra block-relay-only peers. */
-    bool m_initial_sync_finished{false};
+    bool m_initial_sync_finished GUARDED_BY(cs_main){false};
```

Comment:
Apart from the feedback already given, this also seems arbitrary. If you want to document symbols that happen to be under the cs_main scope, why only document this one, and not everything else, like `m_highest_fast_announce` ...

Maybe leave this hunk for a separate pull request? Otherwise I am not sure if this will move forward.

Reply:
`m_highest_fast_announce` is a member of PeerManagerImpl, not CNode or Peer, so is out of scope for this PR...